src/components/TheWelcome.vue

```diff
@@ -1,3 +1,9 @@
 <template>
-  <div>안녕하세요!</div>
+  <div>{{ content }}</div>
 </template>
+
+<script setup lang="ts">
+defineProps(['content']);
+</script>
+
+<style scoped></style>
```

Comment:
저희는 프로젝트마다 다르지만 scss, tailwind를 사용합니다.

```suggestion
<style lang="scss" scoped></style>
```